Description: remove forcefallback
Author: Stephan Lachnit <stephanlachnit@debian.org>
Forwarded: not-needed

--- a/meson.build
+++ b/meson.build
@@ -6,14 +6,10 @@
   default_options: [
     'cpp_std=c++20',
     'warning_level=2',
-    'force_fallback_for=wlroots,libliftoff,vkroots',
   ],
 )
 
 fallbacks = get_option('force_fallback_for')
-if not (fallbacks.contains('wlroots') and fallbacks.contains('libliftoff') and fallbacks.contains('vkroots'))
-  error('!!!"force_fallback_for" is missing entries!!!\n\tPlease do not remove entries from force_fallback_for if you are packaging the project.\n\tWe pull in these projects at specific commits/forks/builds for a reason.\n\tIf you are not packaging, remove this line to continue.')
-endif
 
 add_project_arguments([
   '-DWLR_USE_UNSTABLE',
